libs/filaflat/src/BlobDictionary.h

```diff
@@ -29,8 +29,14 @@ class BlobDictionary {
     BlobDictionary() = default;
     ~BlobDictionary() = default;
 
+    using Blob = std::vector<uint8_t>;
+
     inline void addBlob(const char* blob, size_t len) noexcept {
-        mBlobs.push_back({ blob, len });
+        mBlobs.emplace_back(Blob(blob, blob + len));
```

Comment:
I think you can avoid a move here if you do
```
mBlobs.emplace_back(blob, blob + len)
```

Reply:
Good call. It's actually how `emplace_back` is supposed to be used.

Reply:
Fixed, thanks Ben!